_releaser/main.go

```diff
@@ -29,3 +31,19 @@ func main() {
 		}))
 	ctx.FatalIfErrorf(ctx.Run())
 }
+
+// getEnvOrSecret retrieves secret's value from secret file or env
+func getEnvOrSecret(name string) string {
+	if v, ok := os.LookupEnv(name); ok {
+		return v
+	}
+	secretPath := filepath.Join("/run/secrets", name)
+	if _, err := os.Stat(secretPath); err != nil {
```

Comment:
Guess this one is redundant, as we ignore any error below anyway 😁

Reply:
Ah yes good catch, done.